apps/ima/views/confirmation.html

```diff
@@ -19,8 +20,12 @@ <h2 class="govuk-heading-m">{{#t}}pages.confirmation.header2{{/t}}</h2>
   <p class="govuk-body">{{#t}}pages.confirmation.paragraph2{{/t}}</p>
 
   <h3 class="govuk-heading-m">{{#t}}pages.confirmation.header3{{/t}}</h2>
+    <p class="govuk-body">{{#t}}pages.confirmation.paragraph3{{/t}}</p>
     <p class="govuk-body">{{#t}}pages.confirmation.paragraph4{{/t}}</p>
     <p class="govuk-body">{{#t}}pages.confirmation.paragraph5{{/t}}</p>
+    <p class="govuk-body">{{#t}}pages.confirmation.paragraph6{{/t}}</p>
+    <p class="govuk-body">{{#t}}pages.confirmation.paragraph7{{/t}}</p>
+    <p class="govuk-body">{{#t}}pages.confirmation.paragraph8{{/t}}</p>
 
```

Comment:
I'd put this in a markdown

Reply:
We have now form-submitted.html with markdown. This file is removed.